### DISCUSSION TITLE
用户手册的改版诉求收集

### DISCUSSION THREAD

**Discussion by wenmine** [opentiny/tiny-engine#171] [Ideas]
大家可以在这里提一下对用户手册的诉求，我们这边收集一下，会整体进行评估优先级，感谢大家的支持和帮助(＾Ｕ＾)

**Comment by ChrisLo0751** [comment]
强烈要求，给用户提供一套常规的表单，表格增删改查基本流程的指导教程。
类似这样的非常常规的组件，我觉得是可以把编辑器的主要功能得到体现，这非常绝佳的例子！

**Comment by pangweibao** [comment]
能把导入第三方组件改简单点吗？感觉挺难搞，出现问题不知道怎么解决

**Reply by FairyShuai** [reply]
同感，经常出现物料不显示，我最近导入OpenTiny的Dropdown下拉菜单物料，遇到一大堆问题，顺便问一下，像下拉菜单这种层级结构的物料，应该如何导入
![Uploading 微信图片_20240408135625.png…]()

**Reply by danny00lo** [reply]
同感。

**Reply by wenmine** [reply]
ok，我们看一下哈

**Comment by liaodalin19903** [comment]
建议将tiny-engine的设计思路和架构图进行展现，才能知道是怎么运行的。
